content/kubermatic/master/tutorials_howtos/CCM-CSI_migration/_index.en.md

```diff
@@ -110,3 +129,8 @@ condition `CSIKubeletMigrationCompleted` will be set to true, and the migration
 
 Since the Kubernetes community is on the way to deprecating in-tree CCM, once the `externalCloudProvider` feature gets
 enabled, it cannot be disabled.
+
+[openstack-ccm-reqs]: https://github.com/kubernetes/cloud-provider-openstack/blob/721615aa256bbddbd481cfb4a887c3ab180c5563/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md
```

Comment:
I think only the `Config openstack-cloud-controller-manager` section is important here, so maybe we want to link directly to it.